src/pseudopeople/noise_functions.py

```diff
@@ -357,14 +361,16 @@ def copy_from_household_member(
     :returns: pd.Series where data has been noised with other values from a list of possibilities
     """
 
-    copy_values = data[COPY_HOUSEHOLD_MEMBER_COLS[column_name]]
-    column = pd.Series(copy_values, index=data.index, name=column_name)
-    return column
+    copy_values = data.loc[to_noise_idx, COPY_HOUSEHOLD_MEMBER_COLS[column_name]]
+    data.loc[to_noise_idx, column_name] = ensure_dtype(
```

Comment:
I'd expect the only way this could change the dtype is if the copy column had the wrong dtype. Couldn't we avoid this call if we make sure the copy columns have the correct dtypes after we load the data, since those columns don't receive noise?

Reply:
The "correct" dtype isn't well-defined here, since it depends on the noise order: we want to make sure that the values we copy have the same dtype as this column has _at this point in the noising process_, which is not necessarily the same as its dtype when first loaded nor its dtype when returned to the user.